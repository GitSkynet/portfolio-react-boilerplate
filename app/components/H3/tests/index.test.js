import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import H3 from '../index';

describe('<H3 />', () => {
  it('Should render and match the snapshot', () => {
    const renderedComponent = shallow(<H3 />);
    expect(toJson(renderedComponent)).toMatchSnapshot();
  });

  it('Should adopt the "color" property', () => {
    const color = '#bada55';
    const renderedComponent = shallow(<H3 color={color} />);
    expect(renderedComponent.prop('color')).toEqual(color);
  });
});
