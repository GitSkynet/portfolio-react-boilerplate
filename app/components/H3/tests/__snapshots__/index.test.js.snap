// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H3 /> Should render and match the snapshot 1`] = `
.c0 {
  margin: 0;
  color: inherit;
  font-size: 1.44rem;
  line-height: 1.2;
  -webkit-hyphens: auto;
  -moz-hyphens: auto;
  -ms-hyphens: auto;
  hyphens: auto;
  overflow-wrap: break-word;
  word-wrap: break-word;
  -ms-word-break: break-all;
  word-break: break-word;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
  font-style: normal !important;
  font-weight: 700 !important;
}

<h3
  className="c0"
/>
`;
